src/Pages/Templates/TemplateDetails/components/Tabs/TemplateRepositoriesTab.tsx

```diff
@@ -0,0 +1,171 @@
+    paddingBottom: '8px',
+    color: global_Color_200.value,
+  },
+  mainContainer: {
+    backgroundColor: global_BackgroundColor_100.value,
+    display: 'flex',
+    flexDirection: 'column',
+    height: '100%',
+  },
+  topContainer: {
+    justifyContent: 'space-between',
+    padding: '16px 24px',
+    height: 'fit-content',
+    minHeight: '68px', // Prevents compacting of the search box (patternfly bug?)
+  },
+  bottomContainer: {
+    justifyContent: 'space-between',
+    minHeight: '68px',
+  },
+});
+
+const perPageKey = 'TemplateRepositoriesPerPage';
+
+export default function TemplateRepositoriesTab() {
+  const classes = useStyles();
+  const { templateUUID: uuid } = useParams();
+  const storedPerPage = Number(localStorage.getItem(perPageKey)) || 20;
+  const [page, setPage] = useState(1);
+  const [perPage, setPerPage] = useState(storedPerPage);
+  const [searchQuery, setSearchQuery] = useState('');
+  const [activeSortIndex, setActiveSortIndex] = useState<number>(-1);
+  const [activeSortDirection, setActiveSortDirection] = useState<'asc' | 'desc'>('asc');
+
+  const hasSearch = useMemo(() => !!searchQuery, [searchQuery]);
+  useEffect(() => {
+    setPage(1);
+  }, [hasSearch]);
+
+  const columnSortAttributes = ['repository_name', 'created_at'];
+  const sortString = useMemo(
+    () =>
+      activeSortIndex === -1
+        ? ''
+        : columnSortAttributes[activeSortIndex] + ':' + activeSortDirection,
+    [activeSortIndex, activeSortDirection],
+  );
+
+  const {
+    isLoading,
+    isFetching,
+    isError,
+    error,
+    data = { data: [], meta: { count: 0, limit: 20, offset: 0 } },
+  } = useFetchTemplateSnapshotsQuery(uuid as string, page, perPage, searchQuery, sortString);
+
+  const onSetPage = (_, newPage) => setPage(newPage);
+  const onPerPageSelect = (_, newPerPage, newPage) => {
+    // Save this value through page refresh for use on next reload
+    setPerPage(newPerPage);
+    setPage(newPage);
+    localStorage.setItem(perPageKey, newPerPage.toString());
+  };
+
+  const {
+    data: snapshotList = [],
+    meta: { count = 0 },
+  } = data;
+
+  const fetchingOrLoading = isFetching || isLoading;
+  const loadingOrZeroCount = fetchingOrLoading || !count;
+
+  const sortParams = (columnIndex: number): ThProps['sort'] => ({
+    sortBy: {
+      index: activeSortIndex,
+      direction: activeSortDirection,
+      defaultDirection: 'asc',
+    },
+    onSort: (_event, index, direction) => {
+      setActiveSortIndex(index);
+      setActiveSortDirection(direction);
+    },
+    columnIndex,
+  });
+
+  if (isLoading) {
+    return <Loader />;
+  }
+  if (isError) {
+    throw error;
+  }
+
+  return (
+    <Grid className={classes.mainContainer}>
+      <InputGroup className={classes.topContainer}>
+        <InputGroupItem>
+          <TextInput
+            id='search'
+            ouiaId='name_search'
+            placeholder='Filter by repository name'
```

Comment:
```suggestion
            placeholder='Filter by name'
```

Reply:
This matches elsewhere and prevents concatenation.

Reply:
ok, fixed 👍🏼